lib/queue.js

```diff
@@ -44,6 +44,40 @@ Queue.prototype.list = function(opts, callback) {
 };
 
 /**
+ * Get an individual queue item
+ */
+
+Queue.prototype.item = function(opts, callback) {
+  var arg0 = typeof arguments[0];
+
+  if (arg0 === 'function') {
+    callback = opts;
+    opts = {};
+  } else {
+    if (arg0 === 'string' || arg0 === 'number') {
+      opts = {
+        number: opts
+      };
+    }
+    opts = opts || {};
+  }
```

Comment:
We should check that number is being set and return an error if not. See https://github.com/silas/node-jenkins/blob/cc85733cba9006a58f27a35f68d6e3f99f229789/lib/queue.js#L69-97